config/.stylelintrc.json

```diff
@@ -0,0 +1,91 @@
+{
+   "plugins": [
+      "stylelint-scss",
+      "stylelint-order"
+   ],
+   "rules": {
+      "scss/at-import-no-partial-leading-underscore": true,
+      "scss/at-import-partial-extension": "never",
```

Comment:
This rule causes this line to fail in the template -- https://github.com/Financial-Times/origami-build-tools/blob/ee546af0e0dff8fd5e7dfcb900fa23186663ad1e/templates/component-boilerplate/test-sass-main.js#L9

Reply:
```
Error - Unexpected extension ".test" in @import (scss/at-import-partial-extension)
```